src/modules/contactUs/contact-us.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { ContactUsDto } from './dto/contact-us.dto';
+import * as nodemailer from 'nodemailer';
+import * as validator from 'validator';
+
+@Injectable()
+export class ContactService {
+  private transporter: nodemailer.Transporter;
+
+  constructor() {
+    if (!process.env.EMAIL_USER || !process.env.EMAIL_PASS) {
+      throw new HttpException('Email configuration is missing', HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    this.transporter = nodemailer.createTransport({
+      service: 'Gmail',
+      auth: {
+        user: process.env.EMAIL_USER,
+        pass: process.env.EMAIL_PASS,
+      },
+    });
+  }
+
+  async sendInquiry(contactDto: ContactUsDto): Promise<void> {
+    // Validate email
+    if (!validator.isEmail(contactDto.email)) {
+      throw new HttpException('Invalid email address', HttpStatus.BAD_REQUEST);
+    }
```

Comment:
This sort of validation is not necessary if you validate DTOs with [class validators](https://docs.nestjs.com/techniques/validation)

Reply:
I have made the changes